src/components/AccountModule/connection-hooks.js

```diff
@@ -63,13 +63,26 @@ export function useWalletConnectionDetails(
 ) {
   const { walletNetworkName, hasNetworkMismatch } = useNetworkConnectionData()
   const theme = useTheme()
-  let connectionDetails = {}
   const isWalletAndClientSynced =
     Math.abs(walletSyncDelay - clientSyncDelay) <= OK_PROVIDER_SYNC_DELAY
   const networkSlowdown =
     walletSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     clientSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     isWalletAndClientSynced
+  const networkName = network.shortName.toLowerCase()
```

Comment:
What do you think about using some part of the network config instead? I'd like to move away from anything outside of `chainId` for chain detection, and here it might make more sense to "whitelist" certain chains as being "live".

Reply:
Actually this name is coming from our static network config. :) But should we introduce a `LIVE_CHAINS` map or something similar instead?

Reply:
Yes, either that or mark it in the network configuration that it is / should not be considered as "live".

Reply:
Did the latter in https://github.com/aragon/aragon/pull/1474/commits/bf3f2f869a9313b6edf02e1de4e5d87feb366c93, as we're phasing out `network.type` soon.